platforms/js/opencv_js.config.py

```diff
@@ -74,9 +74,12 @@
         'aruco_Board': ['create'],
         'aruco_GridBoard': ['create', 'draw'],
         'aruco_CharucoBoard': ['create', 'draw'],
+        'aruco_DetectorParameters': ['create']
         }
 
-calib3d = {'': ['findHomography', 'calibrateCameraExtended', 'drawFrameAxes', 'estimateAffine2D', 'getDefaultNewCameraMatrix', 'initUndistortRectifyMap', 'Rodrigues']}
+calib3d = {'': ['findHomography', 'calibrateCameraExtended', 'drawFrameAxes', 'estimateAffine2D', \
+                'getDefaultNewCameraMatrix', 'initUndistortRectifyMap', 'Rodrigues', \
+                'solvePnP', 'solvePnPRansac', 'solvePnPRefineLM']}
```

Comment:
Seems unrelated to this PR

Reply:
It is not a bugfix that's true. However if we enable Aruco markers in the build, we usually would want to use the Aruco board for camera calibration, pose estimation and so on (that's what I am using it for), so I thought it would be straightforward to just add the solvePnP methods to the js build. I could remove it if you want, however, I don't see any major problems in just adding the 3 functions?!